supervisor/api/host.py

```diff
@@ -261,11 +261,12 @@ async def advanced_logs_handler(
                             await response.prepare(request)
                             headers_returned = True
                         await response.write(line.encode("utf-8") + b"\n")
-                    except ClientConnectionResetError as err:
+                    except (ClientConnectionResetError, ConnectionError) as err:
```

Comment:
`ClientConnectionResetError` is subclass of `ConnectionError`, so the tuple is redundant:
```suggestion
                    except ConnectionError as err:
```